packages/@okta/vuepress-site/use_cases/inline_hooks/token_hook/token_hook/index.md

```diff
@@ -296,6 +302,106 @@ Returning an error object will cause Okta to return an OAuth 2.0 error to the re
   ]
 }
 ```
+### Sample Response to Replace an Existing Claim
+
+You can modify existing custom claims or OIDC standard profile claims, such as `birthdate` and `locale`. You can't, however, modify any system-specific claims, such as `iss` or `ver`, and you can't modify a claim that isn't currently part of the token in the request payload. Attempting to modify a system-specific claim or using an invalid operation results in the entire PATCH failing and errors logged in the token hooks events.
+
+For the list of access token reserved claims that you can't modify, see [Access Tokens Scopes and Claims](https://developer.okta.com/docs/api/resources/oidc/#access-token-scopes-and-claims). Note that although the `aud` claim is listed as a reserved claim, you can modify that claim in a response.
+
+See [ID Token Claims](https://developer.okta.com/docs/api/resources/oidc/#id-token-claims) for a list of ID token reserved claims that you can't modify.
+
+```json
+{
+   "commands": [ 
+        { 
+            "type": "com.okta.identity.patch",
+            "value": [ 
+                 { 
+                     "op": "replace",
+                     "path": "/claims/extPatientId",
+                     "value": "1234"
+                  }
+             ] 
+         }, 
+         { 
+             "type": "com.okta.access.patch",
+             "value": [
+                  { 
+                     "op": "replace",
+                     "path": "/claims/external_guid",
+                     "value": "F0384685-F87D-474B-848D-2058AC5655A7" 
+                   }
+              ] 
+          }
+      ]
+}
+```
+
+### Sample Response to Modify Token Lifetime
+You can modify how long the Access and ID tokens are valid by specifying the `lifetime` in seconds. The `lifetime` value must be a minimum of five minutes (300 seconds) and a maximum of 24 hours (86,400 seconds).
```

Comment:
`for how long`, same as here https://github.com/okta/okta-developer-docs/pull/375/files#r287127334. I could be wrong though and `for` isn't necessary.